### PR TITLE
Adding contact specialist link to bnmo artwork

### PR DESCRIPTION
**Pull request by oxaudo**
Can't figure out just yet where the email to contact a specialist is set. For now just reusing the logic that we currently have on auction page to contact a specialist. This logic seem to be using inquiry decision tree that is quite complex. Don't feel like I know enough to change any of that though...

The proposed solution seem to do what https://artsyproduct.atlassian.net/browse/DISCO-226 asks for except for not modifying any emails for now. Will try to figure that out next of maybe somebody know right away: @sweir27 or @damassi ?

### CODE REVIEW

File: src/desktop/apps/artwork/components/commercial/templates/index.jade [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- var newBuyNowFlow = user && user.hasLabFeature("New Buy Now Flow")
```
Comment (oxaudo) [inline src/desktop/apps/artwork/components/commercial/templates/index.jade]: We'll need to reuse it in both acquire and questions partials so moving it up here.

In general all of those partials are only seem to be used in this one spot below, so not sure how much we gain with all the partials break up here.

File: src/desktop/apps/artwork/components/commercial/templates/questions.jade [hunk 2]
```diff
@@ -1,6 +1,9 @@
 if artwork.is_for_sale
   .artwork-commercial__questions
-    | Questions about buying art on Artsy?
-    | Read our&nbsp;
-    a.collector-faq( href='/#' ) Collector FAQ
+    | Have a question?&nbsp;
+    a.collector-faq( href='/#' ) Read our FAQ
+    if (artwork.is_acquireable && (newBuyNowFlow && artwork.partner.type != 'Auction House'))
```
Comment (oxaudo) [inline src/desktop/apps/artwork/components/commercial/templates/questions.jade]: This seem to be a current check for the proper 'Buy now' artwork

File: src/desktop/components/inquiry_questionnaire/map/decisions.coffee [hunk 3]
```diff
@@ -5,7 +5,7 @@ hasSeenThisSession = (steps...) -> ({ logger }) -> logger.hasLoggedThisSession s
 
 decisions =
   ask_specialist: ({ artwork, ask_specialist }) ->
-    artwork.get('is_in_auction') is true || ask_specialist is true
+    artwork.get('is_in_auction') is true || ask_specialist is true || artwork.get('ecommerce') is true
```
Comment (oxaudo) [inline src/desktop/components/inquiry_questionnaire/map/decisions.coffee]: Here is where we seem to decide what dialog to display. Not sure if I should have used the same `is_acquireable` logic here but `ecommerce` seem to be properly set on artwork at that point.

**Comment by oxaudo** [comment]
screen shot 2018-09-27 at 1 59 14 pm

File: src/desktop/components/inquiry_questionnaire/map/decisions.coffee (hunk 3 above)
Comment (damassi) [reply src/desktop/components/inquiry_questionnaire/map/decisions.coffee]: Could also do `artwork.get('is_in_auction') || artwork.get('ecommerce') || ask_specialist`

**Comment by oxaudo** [comment]
Updated the line for consignment as well. Here is how it looks in case of multiple artists:
screen shot 2018-09-27 at 2 48 33 pm

In case of single artist on the work I still kept the artist name here. @briansw, @stephen-i-chan - that is a bit different than what you have in a design. Do we want to remove artist name in this prompt even if it's only one artist or keep at as I currently do?
screen shot 2018-09-27 at 2 50 44 pm

File: src/desktop/apps/artwork/lib/inquire.coffee [hunk 4]
```diff
@@ -4,7 +4,7 @@ Artwork = require '../../../models/artwork.coffee'
 ArtworkInquiry = require '../../../models/artwork_inquiry.coffee'
 openInquiryQuestionnaireFor = require '../../../components/inquiry_questionnaire/index.coffee'
 
-module.exports = (id) ->
+module.exports = (id, ask_specialist = false) ->
```
Comment (oxaudo) [inline src/desktop/apps/artwork/lib/inquire.coffee]: @sweir27 - passing this `ask_specialist` as on optional variable here to pass through to the `openInquiryQuestionnaireFor`. Does look cleaner I think. Thanks for pointing me to that option. 

How does that look to you?

**Comment by oxaudo** [comment]
I think this is good to merge unless somebody has objections. @sweir27 or @damassi?

**Pull request merged**